### DISCUSSION TITLE
How do i authenticate with app store when my account is using a "Managed Apple ID"

### DISCUSSION THREAD

**Discussion by johanhelsing-attensi** [fastlane/fastlane#18592] [Q&A]
When I'm trying to set up fastlane match, I get an error:

```
[!] The request could not be completed because:
        Access forbidden
```

Now my apple account is set up so that when I log in to app store connect, I'm redirected through login.microsoftonline.com, so I'm wondering if regular password login probably doesn't work for me.

So I searched a bit around and it seems like application specific password is the way to go... however, I don't have the option to generate those in the security section on https://appleid.apple.com/account/manage . Searching for that it looks like you need to turn on 2FA, but I don't see that either, just this message:

![image](https://user-images.githubusercontent.com/80466796/115849755-7c3c3d80-a425-11eb-8b2a-0329e2454594.png)

Now I also found this from a year ago saying app specific passwords just arent supported with managed apple ids: https://developer.apple.com/forums/thread/128251

So maybe it's not possible to use fastlane match with a managed apple id?

Can I bypass some steps downloading the certificates manually instead? Or am I out of luck?

**Comment by dinsen** [comment]
Hi @johanhelsing-attensi 
Is it possible for you to generate an API key? 
https://docs.fastlane.tools/app-store-connect-api/
